README.md

```diff
@@ -139,7 +151,10 @@ EDQueueJobDidFail
 ---
 
 ### iOS Support
-EDQueue is designed for iOS 5 and up.
+EDQueue is designed for iOS 7 and up.
 
 ### ARC
 EDQueue is built using ARC. If you are including EDQueue in a project that **does not** use [Automatic Reference Counting (ARC)](http://developer.apple.com/library/ios/#releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html), you will need to set the `-fobjc-arc` compiler flag on all of the EDQueue source files. To do this in Xcode, go to your active target and select the "Build Phases" tab. Now select all EDQueue source files, press Enter, insert `-fobjc-arc` and then "Done" to enable ARC for EDQueue.
+
+### Nullability
```

Comment:
I'd like to mention somehow the codebase grows to be 100% swift compatible. Maybe better title? Or mention compatibility somewhere else in readme?